.travis.yml

```diff
@@ -1,13 +1,24 @@
+sudo: required
+dist: trusty
+group: edge
+
 language: go
 go:
     - 1.6.x
     - 1.7.x
     - 1.8.x
     - tip
+
 install:
     - go get -v golang.org/x/tools/cmd/cover
+    - sudo add-apt-repository -y ppa:opencpu/opencpu-1.6
```

Comment:
Yes I'm sorry about that, the single user server API has indeed changed in v2.0. Does the new API not work for you/